CK2Plus_expanded/history/titles/k_burgundy.txt

```diff
@@ -77,13 +77,13 @@
 	law = agnatic_succession
 }
 
-#768.9.24 = {
+#768.9.24 = { # added by history effect, to avoid wrong display on character selection
```

Comment:
Where is the history effect? I don't see it.

Reply:
old_frankish.txt

Reply:
Was it done in another commit somewhere else that's not part of this PR? I don't see that file in here anywhere.

Reply:
Nevermind, found it